.github/workflows/test-pull-request.yml

```diff
@@ -3,7 +3,17 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - name: Check out code
+      uses: actions/checkout@master
+    - name: Login to docker hub
+      if: success()
+      uses: actions-hub/docker/login@master
+      env:
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
     - uses: OrangeLabs-moe/gradle-actions@v5.0-openjdk-13
+      env:
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       with:
-        args: test
+        args: test jib
```

Comment:
`jib` should only run on push for master - https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions